### PR TITLE
Update ARM sources

### PR DESCRIPTION
**Pull request by OniriCorpe**
## Problem

- the arm sources are really really outsdated

## Solution

- replaced it with this one https://github.com/remiheens/mattermost-docker-arm

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by kemenaran** [comment]
LGTM. The current armv7 sources are indeed heavily deprecated (because maintaining the build script to accommodate changes upstream is hard), so switching to more up-to-date sources sounds much better.

File: manifest.toml [hunk 1]
```diff
@@ -7,7 +7,7 @@ description.fr = "Plateforme de collaboration open source conçue pour les déve
 
 version = "9.4.2~ynh1"
 
-maintainers = ["pmorinerie"]
+maintainers = [ "pmorinerie" ]
```
Comment (kemenaran) [inline manifest.toml:10]: Could you remove unrelated changes from the toml, like the extra spaces and indentation changes ?

**Review by kemenaran — CHANGES_REQUESTED** [review]

File: manifest.toml (hunk 1 above)
Comment (OniriCorpe) [reply manifest.toml:10]: I’m sorry it’s my automatic formatter

Comment (OniriCorpe) [reply manifest.toml]: done

**Review by kemenaran — APPROVED** [review]

**Comment by kemenaran** [comment]
Thanks!
